src/ui/orderandprops.cc

```diff
@@ -33,15 +33,15 @@ bool dictLessThan( sptr< Dictionary::Class > const & dict1, sptr< Dictionary::Cl
   int idFrom1 = dict1->getLangFrom();
   int idTo1   = dict1->getLangTo();
   if ( idFrom1 == 0 ) {
-    QPair< quint32, quint32 > ids = LangCoder::findIdsForName( QString::fromUtf8( dict1->getName().c_str() ) );
+    QPair< quint32, quint32 > ids = LangCoder::findLangIdPairFromStr( QString::fromUtf8( dict1->getName().c_str() ) );
```

Comment:
can ` QString::fromUtf8( dict1->getName().c_str() ) ` be 
replaced by `dict1->getName()` directly

Reply:
Not really. `Dictionary::dictionaryName` is `std::string.`

`QRegularExpression` only accepts `QString`. The conversion needs to happen.